lib/node/fs.dart

```diff
@@ -6,15 +6,22 @@ import 'dart:js_interop';
 
 import 'package:js/js.dart' as js;
 
+/// POSIX functions for interacting with the file system.
+/// Wraps https://nodejs.org/api/fs.html
 @JS()
 external FileSystem get fs;
 
 @JS()
 @js.staticInterop
-class FileSystem {}
+inline class FileSystem {
+  final JSObject fileSystem;
+  FileSystem(this.fileSystem);
 
-extension FileSystemExtension on FileSystem {
+  @JS()
```

Comment:
In general, you shouldn't need the `@JS` on inline class external members unless you're doing renaming. It doesn't hurt, of course.

Reply:
Talked offline: some `@JS` annotations are needed because our static error checking incorrectly returns errors without it sometimes. Resolving.